From 6a98af52dae360d5a9aaede3073365ff8cdff933 Mon Sep 17 00:00:00 2001
From: rajucm <raju.mallikarjun.chegaraddi@intel.com>
Date: Tue, 16 Nov 2021 11:37:17 +0530
Subject: [PATCH] [sensor-hal] Isonsor poll re-entry causes issue

poll re-entry is not expected in passthrogh implementation.
we are avoiding by comment

Tracked-On:
Signed-off-by: rajucm <raju.mallikarjun.chegaraddi@intel.com>
---
 sensors/1.0/default/Sensors.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sensors/1.0/default/Sensors.cpp b/sensors/1.0/default/Sensors.cpp
index 1100dd6a2..700ae84b6 100644
--- a/sensors/1.0/default/Sensors.cpp
+++ b/sensors/1.0/default/Sensors.cpp
@@ -166,7 +166,7 @@ Return<void> Sensors::poll(int32_t maxCount, poll_cb _hidl_cb) {
     int err = android::NO_ERROR;
 
     { // scope of reentry lock
-
+#if 0
         // This enforces a single client, meaning that a maximum of one client can call poll().
         // If this function is re-entred, it means that we are stuck in a state that may prevent
         // the system from proceeding normally.
@@ -182,7 +182,7 @@ Return<void> Sensors::poll(int32_t maxCount, poll_cb _hidl_cb) {
                     "ISensors::poll() re-entry. I do not know what to do except killing myself.";
             ::exit(-1);
         }
-
+#endif
         if (maxCount <= 0) {
             err = android::BAD_VALUE;
         } else {
-- 
2.33.0

